services-manager/src/main/resources/application-example.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 Crown Copyright
```

Comment:
Is there a reason to have example, services and jvm profiles? can they not be merged?

Reply:
I like that they are separate profiles.
Example - adds in the example jars so those beans can be used and example cache warming used
services - starts up the palisade services
discovery - starts up the eureka services
So unless you can create a profile jvm_example that just calls to the other profiles, I would leave this as is.

Reply:
Leaving this as is for now.

Reply:
Addressed in PAL-606 or 607... whatever I'm doing